requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pytest==3.10.1
+#pytest==3.10.1
```

Comment:
So the solution is commenting the whole line? If that's so, that we can just remove it. Otherwise we should have there just 'pytest'